background/usercss-install-helper.js

```diff
@@ -96,28 +96,35 @@ bgReady.all.then(() => {
     const u = new URL(url);
     const m = maybeDistro[u.hostname];
     if (!m || m.rx.test(u.pathname)) {
-      openInstallerPage(tabId, url, {});
+      openInstallerPage(tabId, url, {}).catch(console.error);
       // Silently suppress navigation.
       // Don't redirect to the install URL as it'll flash the text!
-      return {redirectUrl: 'javascript:void 0'}; // eslint-disable-line no-script-url
+      return {cancel: true};
     }
   }
 
-  function openInstallerPage(tabId, url, {code, inTab} = {}) {
+  async function openInstallerPage(tabId, url, {code, inTab} = {}) {
     const newUrl = `${URLS.installUsercss}?updateUrl=${encodeURIComponent(url)}`;
     if (inTab) {
-      browser.tabs.get(tabId).then(tab =>
-        openURL({
-          url: `${newUrl}&tabId=${tabId}`,
-          active: tab.active,
-          index: tab.index + 1,
-          openerTabId: tabId,
-          currentWindow: null,
-        }));
-    } else {
-      const timer = setTimeout(clearInstallCode, 10e3, url);
-      installCodeCache[url] = {code, timer};
-      chrome.tabs.update(tabId, {url: newUrl});
+      const tab = browser.tabs.get(tabId);
+      return openURL({
+        url: `${newUrl}&tabId=${tabId}`,
+        active: tab.active,
+        index: tab.index + 1,
+        openerTabId: tabId,
+        currentWindow: null,
+      });
+    }
+    const timer = setTimeout(clearInstallCode, 10e3, url);
+    installCodeCache[url] = {code, timer};
+    try {
+      await browser.tabs.update(tabId, {url: newUrl});
+    } catch (err) {
+      // FIXME: https://github.com/openstyles/stylus/issues/1367
```

Comment:
It's very inconvenient when a comment is a riddle that needs you to open an external page with a lot of comments and spend time to make sense of it.

Since you've used FIXME, it means a better fix is needed or we need to revisit this when Kiwi fixes itself. In that case write it explicitly. If it's just informational then use `Kiwi bug workaround` instead of FIXME.